packages/ui/src/components/ProTable/types.ts

```diff
@@ -63,7 +63,7 @@ export interface ProColumnType<T = any> extends ColumnType<T> {
     title: ReactNode;
     icon?: ReactElement;
     iconTitle?: ReactNode;
-    tooltip?: string;
+    tooltip?: ReactNode;
```

Comment:
tooltip in proTable is used for antd Tooltip title
as the type expected by antd, (`title?: React.ReactNode | RenderFunction;`) tooltip here should be ReactNode